functions/src/users/index.ts

```diff
@@ -13,15 +16,15 @@ const validateUser = (value: IUser): Promise<void> => {
 
 export const triggerEventsWhenUserIsCreated = functions.firestore
   .document('users/{userId}')
-  // eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
   .onCreate((snapshot: DocumentSnapshot, context: EventContext) => {
-    return validateUser(snapshot.data() as IUser).catch(errors => {
-      console.error('Invalid User Found: ', errors);
-      return firestore()
-        .collection('users')
-        .doc(context.params.userId)
-        .delete();
-    });
+    return validateUser(snapshot.data() as IUser)
+      .catch(errors => {
+        console.error('Invalid User Found: ', errors);
+        return db
+          .collection('users')
+          .doc(context.params.userId)
+          .delete();
+      });
```

Comment:
just a formatting thing, but we should always put the `.then` `.catch` or `.finally` on new lines so the reader can see which promise resolution this block is for.

Reply:
okay